CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 0.19.0 (April 22, 2025)
+
+NOTES:
+
+* **This release requires Juju controller version 2.9.49 or higher juju.**
+* **If using JAAS, this release requires Juju controller version 3.5.0 or higher.**
+* This release uses juju client api code from the juju 3.6.3 release.
```

Comment:
This should say juju 3.6.4 instead.
And there is a mix of "juju" and "Juju" that could be fixed.

Reply:
done, thanks